### DISCUSSION TITLE
Is the default terminal for Mac OS supported

### DISCUSSION THREAD

**Discussion by KAEYL98** [JanDeDobbeleer/oh-my-posh#2410] [Troubleshoot] [answered]
The shell isn't displaying correctly on the default terminal but looks fine in VS code
<img width="570" alt="Screen Shot 2022-06-16 at 9 04 14 AM" src="https://user-images.githubusercontent.com/42574783/173969766-5e4f9efc-75a3-45e7-95e0-7bd2d9d297c7.png">
.

**Comment by JanDeDobbeleer** [comment, marked as answer]
No it's not. I can't figure out the actual setting that influences this.
